services/QuillLMS/client/app/bundles/Diagnostic/actions/titleCards.ts

```diff
@@ -12,10 +12,11 @@ function startListeningToTitleCards() {
 function loadTitleCards(): (any) => void {
   return (dispatch) => {
     TitleCardApi.getAll(DIAGNOSTIC_TITLE_CARD_TYPE).then((body) => {
-      const titleCards = body.title_cards.reduce((obj, item) => {
+      let titleCards = body.title_cards || []
```

Comment:
The test uses an empty mock for `body`. Thus, `body.<undefined>.reduce()` yields a TypeError in Node 16

Reply:
Take or leave this, but could we get away with safe navigation?  `const titleCards = body?.title_cards?.reduce(...) || {}`  I don't know if that's actually better, but it is a pattern we use fairly often in Ruby.